package.json

```diff
@@ -90,7 +90,9 @@
         "translate:upload": "crowdin upload sources -b crowdin",
         "deploy": "npm run build && gh-pages -d public -b master",
         "deploy-dev": "npm run build && gh-pages -d public",
-        "deploy-branch": "PATH_PREFIX=/deriv-com/br/$branch_name npm run build && gh-pages -d public -e br/$branch_name"
+        "deploy-branch": "PATH_PREFIX=/deriv-com/br/$branch_name npm run build && gh-pages -d public -e br/$branch_name",
+        "prepare": "husky install",
```

Comment:
do we need to husky install everytime?

Reply:
It seems to be a recommendation from their docs. More towards for anyone new who clones and installs the project.
![image](https://user-images.githubusercontent.com/72253841/124696472-ba6ed680-df17-11eb-81d7-eaff62595658.png)